recipes/emcee/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "2.1.0" %}
+
+package:
+  name: emcee
+  version: {{ version }}
+
+source:
+  fn: emcee-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/de/00/2358f12c98fa74ab58d78e6a4a4f9a8152fadf6ade2d809d98d771dcc31b/emcee-{{ version }}.tar.gz
+  md5: c6b6fad05c824d40671d4a4fc58dfff7 
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record=record.txt
+
+requirements:
+  build:
+    - python
+    - numpy
+
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - emcee
+  requires:
+    - nose
+  commands: 
+    - nosetests {{ environ.SRC_DIR }}/emcee -sv
+
+about:
+  home: http://danfm.ca/emcee/
+  license: GNU General Public License (GPL)
```

Comment:
The license in [their setup.py says MIT] (https://github.com/dfm/emcee/blob/master/setup.py#L48).  I am not sure what commit corresponds to the 2.1.0 release though, since that is not tagged on github